app/_locales/am/messages.json

```diff
@@ -181,6 +181,10 @@
   "createPassword": {
     "message": "የይለፍ ቃል ፍጠር"
   },
+  "crypto": {
+    "message": "ክሪፕቶ",
+    "description": "Exchange type"
+  },
```

Comment:
Curious where the translations come from.  My understanding was that devs author the english ones, and a translation services adds the other languages.